cmd/eksctl-anywhere/cmd/root.go

```diff
@@ -64,3 +64,9 @@ func initLogger() error {
 func Execute() error {
 	return rootCmd.ExecuteContext(context.Background())
 }
+
+// RootCmd returns the eksctl-anywhere root cmd with autuGenTag disabled.
+func RootCmd() *cobra.Command {
```

Comment:
i initially do not want to make rootCmd public so have this getter function for other pkg to access. But yea it makes sense since the DisableAutoGenTag is irrelevant to eks-a cmd, and should only be used in the doc gen section, will make the update